packages/toolpad-components/src/Link.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Link as MuiLink, LinkProps as MuiLinkProps } from '@mui/material';
+import { createComponent } from '@mui/toolpad-core';
+
+interface LinkProps extends Omit<MuiLinkProps, 'children'> {
+  href: string;
+  value: string;
+}
+
+function Link({ value, href }: LinkProps) {
+  return (
+    <MuiLink href={href} target="_blank">
```

Comment:
Should we add `rel="noopener noreferrer"` for external links?

Reply:
`nofollow` might be useful too, so that people don't use Toolpad to generate backlinks to their site.